book/02-git-basics/sections/undoing.asc

```diff
@@ -79,7 +79,7 @@ The `CONTRIBUTING.md` file is modified but once again unstaged.
 
 [NOTE]
 =====
-While `git reset` _can_ be a dangerous command if you call it with `--hard`, in this instance the file in your working directory is not touched.
+While `git reset` _can_ be a dangerous command if you call it with `--hard` as it changes files in your working directory, however in the above example the file in your working directory is not touched.
```

Comment:
I think when reading the whole sentence, it is possible that readers get confused about the usage of the phrase "in this instance" thinking that it is used for `git reset --hard` but the intention is the example before that. So using words like "in above example" or "in the instance above" would make it much clearer.